### DISCUSSION TITLE
OMR Router Upgrade

### DISCUSSION THREAD

**Discussion by JRTechServices** [Ysurac/openmptcprouter#3211] [General]
Can sysupgrade be used to update the router or will things break?

**Comment by Ysurac** [comment]
sysupgrade is always used to update the router.

**Comment by pierrejego** [comment]
Hello and thank you for all this work.
I am searching documentation to upgrade from 0.59.1-5.4 to version 0.60

Following this https://github.com/Ysurac/openmptcprouter/issues/2961, I understood I'll have to update VPS, but before I tried to update Routeur with page /cgi-bin/luci/admin/system/sysupgrade

Result is : `**Device not sysupgrade compatible!**`

I have no error in any logs.
Reponse from openserver download seems ok 

`{"images":[{"type":"sysupgrade","name":"","version":"v0.60"}],"bin_dir":"v0.60\/rpi4\/targets\/bcm27xx\/bcm2711"}
`
Dir seems to be present  
`https://download.openmptcprouter.com/release/v0.60/rpi4/targets/bcm27xx/bcm2711/`

but lastet version from issue 2961 link to 
`https://download.openmptcprouter.com/release/v0.60rc2-6.1/rpi4/targets/bcm27xx/bcm2711/`
bu it's a RC so I think it's normal

I found this old issues https://github.com/Ysurac/openmptcprouter/issues/1932  but no solution 

Sorry if it's a stupid question, but shall I update VPS first to be able to use the sysupgrade ?

I will make a fresh install otherwise, I juste wanted to understand if there were special steps to do to update. 

Regards

**Reply by Ysurac** [reply]
Download the image from https://www.openmptcprouter.com/download and upgrade via System->Flash firmware.
I have to fix Sysupgrade.
